windows-wwiv-update/Form2.cs

```diff
@@ -200,16 +202,44 @@ void m_oWorker_DoWork(object sender, DoWorkEventArgs e)
                 // Update Progress Bar
                 m_oWorker.ReportProgress(60);
 
+                // Create Temp Update Directory
+                Directory.CreateDirectory("update-temp");
+
+                // Create Cleanup Batch File If Not Existant
+                string cleanupFile = Path.Combine(Application.StartupPath, "cleanup.bat");
+                if (!File.Exists(cleanupFile))
+                {
+                    File.Delete(cleanupFile);
```

Comment:
It's was 3AM when I wrote that and I'm not sure why I'm deleting the file if it doesn't exit.. Also I will make it a cmd. The idea is this file will be written to perform other clean-ups.. There are unused files in people dirs that are old and outdated such as update.bat - We can use this as well to help clean up those inconsistencies.